test/cases/testdata/reachable/test-reachable-paths-1022.yaml

```diff
@@ -0,0 +1,128 @@
+  cases:
+  - data: {}
+    input_term: '{
+        "graph": {
+          "one": ["two","five"],
+          "two": ["four"],
+          "three": [""],
+          "four": ["three"],
+          "five": ["seven","six"],
+          "six": ["nine"],
+          "seven": ["eight"],
+          "eight": [""],
+          "nine": [""],
+        },
+        "initial": {"one"}
+      }'
+    modules:
+      - |
+        package reachable
+
+        p = result {
+          graph.reachable_paths(input.graph, input.initial, result)
+        }
+    note: reachable_paths/cycle
+    query: data.reachable.p = x
+    want_result:
+      - x:
+          - - one
+            - five
+            - seven
+            - eight
+
+          - - one
+            - five
+            - six
+            - nine
+
+          - - one
+            - two
+            - four
+            - three
+
+  - data: {}
+    input_term: '{
+        "graph": {
+          "one": ["two","five"],
+          "two": ["four"],
+          "three": [""],
+          "four": ["three"],
+          "five": ["seven","six"],
+          "six": ["nine"],
+          "seven": ["eight"],
+          "eight": [""],
+          "nine": [""],
+        },
+        "initial": {"one"}
+      }'
+    modules:
+        - |
+          package reachable
+
+          p = result {
+            graph.reachable_paths(input.graph, input.initial, result)
+          }
+    note: reachable_paths/cycle
```

Comment:
Please use unique `note` values, it'll help debugging later on.

Reply:
thanks, will do